src/Resources/contao/languages/de/tl_content.php

```diff
@@ -13,7 +13,7 @@
 $GLOBALS['TL_LANG']['tl_content']['sliderSlidesPerView'] = ['Anzahl der sichtbaren Slides',"Die Anzahl an einzelnen Slides die pro Sektion sichtbar sein soll (Zahl oder 'auto')."];
 $GLOBALS['TL_LANG']['tl_content']['sliderSpaceBetween'] = ['Abstand zwischen den Slides','Abstand zwischen den Slides in Pixel.'];
 $GLOBALS['TL_LANG']['tl_content']['sliderEffect'] = ['Effekt','Effekt-Art beim Wechsel der Slides.'];
-$GLOBALS['TL_LANG']['tl_content']['sliderWrapperClass'] = ['Wrapper CSS Klasse','Die optionale CSS Klasse des Wrappers.'];
+$GLOBALS['TL_LANG']['tl_content']['sliderWrapperClass'] = ['Wrapper CSS Klasse','Die optionale CSS Klasse des Wrappers. Kann verwendet werden, wenn man z.B. durch eine Nachrichtenliste sliden möchte. Dann muss die Klassen des Listen-Containers angegeben werden (mod_newslist).'];
```

Comment:
Can you add this in en/tl_content.php too please? ;)

Reply:
Fixed a typo in the german translation and added this to the english one.